server/sonar-server/src/main/java/org/sonar/server/platform/cluster/ClusterProperties.java

```diff
@@ -25,6 +25,7 @@
 import org.sonar.api.config.PropertyDefinition;
 
 public class ClusterProperties {
+  // TODO TO BE REMOVED
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)